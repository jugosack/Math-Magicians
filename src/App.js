/* eslint-disable import/no-extraneous-dependencies */
import './App.css';
import { Route, Routes } from 'react-router-dom';
import Navbar from './Navbar';
import Home from './pages/Homepage';
import CalculatorPage from './pages/Calculatorpage';
import Quote from './pages/Quotepage';

function App() {
  return (
    <>
      <Navbar />
      <div className="container_1">
        <Routes>
          <Route path="/" element={<Home />} />
          <Route path="/calculatorpage" element={<CalculatorPage />} />
          <Route path="/qoutepage" element={<Quote />} />
        </Routes>
      </div>
    </>
  );
}

export default App;
