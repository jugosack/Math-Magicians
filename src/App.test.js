import React from 'react';
import { BrowserRouter } from 'react-router-dom';
// eslint-disable-next-line import/no-extraneous-dependencies
import renderer from 'react-test-renderer';
import App from './App';

const MockHeader = () => (
  <BrowserRouter>
    <App />
  </BrowserRouter>
);

test('renders correctly', () => {
  const tree = renderer
    .create(<MockHeader />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
