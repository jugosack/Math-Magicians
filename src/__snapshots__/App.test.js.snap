// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
Array [
  <nav
    className="nav"
  >
    <a
      className="site-title"
      href="/"
      onClick={[Function]}
    >
      Math Magicians
    </a>
    <ul>
      <li
        className="active"
      >
        <a
          href="/"
          onClick={[Function]}
        >
          Home
        </a>
      </li>
      <span
        className="divider"
      />
      <li
        className=""
      >
        <a
          href="/calculatorpage"
          onClick={[Function]}
        >
          Calculator
        </a>
      </li>
      <span
        className="divider"
      />
      <li
        className=""
      >
        <a
          href="/qoutepage"
          onClick={[Function]}
        >
          Qoute
        </a>
      </li>
    </ul>
  </nav>,
  <div
    className="container_1"
  >
    <section
      className="home"
    >
      <h1>
        Welcome to our page!
      </h1>
      <p>
        
      "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum."
      
      </p>
      <p>
        
      "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum."
      
      </p>
    </section>
  </div>,
]
`;
